suite-common/wallet-types/src/coinjoin.ts

```diff
@@ -26,6 +26,7 @@ export interface CoinjoinSession extends CoinjoinSessionParameters {
     timeCreated: number; // timestamp when was created
     timeEnded?: number; // timestamp when was finished
     paused?: boolean; // current state
+    interrupted?: boolean; // it was paused by force
```

Comment:
feel the force Luke :)
https://starwarsblog.starwars.com/wp-content/uploads/2016/06/yoda-and-luke-dagobah-1536x864-926700358605.jpg